From 2c2f59489605a6ea311aa600e35c927346a845a3 Mon Sep 17 00:00:00 2001
From: Nehal J Wani <nehaljw.kkd1@gmail.com>
Date: Sat, 25 Nov 2017 07:39:25 -0800
Subject: [PATCH] Make ncdump_tst_h_rdc0 test conditional

The test depends on files generated by tst_netcdf4, which, at the
moment, is defined only if HAVE_BASH is true.
---
 ncdump/CMakeLists.txt | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/ncdump/CMakeLists.txt b/ncdump/CMakeLists.txt
index 4f0aa12..efc586d 100644
--- a/ncdump/CMakeLists.txt
+++ b/ncdump/CMakeLists.txt
@@ -186,7 +186,9 @@ ENDIF()
     # is different, but the result is actually correct.
     #IF(NOT MSVC)
     add_sh_test(ncdump tst_netcdf4)
-    add_bin_test(ncdump tst_h_rdc0)
+    if(TARGET ncdump_tst_netcdf4)
+      add_bin_test(ncdump tst_h_rdc0)
+    ENDIF()
     #ENDIF()
 
     add_bin_test(ncdump tst_unicode)
-- 
2.10.1

